src/convertToObject.js

```diff
@@ -3,16 +3,36 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssPropetiesArray = sourceString.split(';');
+
+  const reduceCallback = (cssPropetiesObject, properties) => {
+    const arr = properties.split(':');
+
+    if (arr.length < 2) {
+      return cssPropetiesObject;
```

Comment:
I think we dont need this, because we always have only 2 elements in array

Reply:
I guess we need this for empty key or value

Reply:
we dont need this

Reply:
![image](https://user-images.githubusercontent.com/92088873/185895121-d277905d-11af-4939-9bc2-0dbbe8ba1fbe.png)